tokens/properties/alias.json

```diff
@@ -422,7 +422,8 @@
                     "value": "{box-shadow.inset.style.thin.value} {color.utility.red.50.value}"
                 },
                 "button-primary": {
-                    "value": "{box-shadow.inset.style.thick.value} {color.base.100.value}"
+                    "value": "{box-shadow.inset.style.thick.value} {color.base.100.value}",
+                    "comment": "Only applies to filled primary buttons"
```

Comment:
Is this comment meant to be shown in Storybook?

Reply:
Nope!